README.md

```diff
@@ -366,40 +367,69 @@ There is also the custom wipe mode. This mode provides the possibility to addres
 In the case of custom wipe, the factory reset (initramfs context) calls `/opt/factory_reset/custom-wipe` before re-creating the filesystems inside the partitions `etc` and `data`.
 In order to establish the custom wipe mode, a Yocto recipe `omnect-os-initramfs-scripts.bbappend` has to be supplied, which has to install the required utilities.
 
-The factory reset provides the option to exclude particular files or directories.
-For example, it may make sense to keep the WIFI configuration, in order to prevent loosing the network connectivity.
-For this purpose, the OS bootloader environment variable `factory-reset-restore-list` has to be used for.
-In the following example, the regular file `/etc/wpa_supplicant.conf` and the directory
-`/etc/aziot/identityd/` survives the factory reset:<br>
-```sh
-sudo bootloader_env.sh set factory-reset-restore-list '/etc/wpa_supplicant.conf;/etc/aziot/identityd/'
+The factory reset provides the option "preserve" to exclude particular files or directories.
+The topics in the array "preserve" are defined by the keys of [`/etc/omnect/factory-reset.json`](recipes-omnect/omnect-device-service/omnect-device-service/factory-reset.json).<br>
+Furthermore the topic "applications" preserves custom
+configurations defined by configurations in `/etc/omnect/factory-reset.d`.
+#### Custom Factory Reset Configuration
+A custom configuration to preserve files from a factory reset is a json file in `/etc/omnect/factory-reset.d` with the format:<br>
+
+```json
+{
+  "paths": [
+    "/home/omnect/.bash_history",
+    "/home/root/.bash_history"
+  ]
+}
 ```
+This example preserves the `bash_history` of the users `omnect` and `root`.
 
-The list of path names is separated by the character `;` and is enclosed by the `'` quotation mark.
-The factory reset is directed to the partitions `etc` and `data`.
-Therefore, path names with the following prefixes are allowed: `/etc/`, `/home/`, `/var/lib/`, `/var/log/`, `/usr/local/` and /mnt/data.
-
-In the case of an error during the backup of a file or directory part of the restore list, the whole factory reset will be aborted
+#### Factory Reset Result
+In the case of an error during the backup of a files or directories the whole factory reset will be aborted
 and the partitions `etc` and `data` remain untouched.

```

Comment:
is it really important? it will work with /home/ or /mnt/data/home/ ... the mechanism doesn't distinguish.
it will fail if the path you use isn't overlayed though.
do you have a proposal?

Reply:
maybe something like "Note that home directories in example above work because they are actually located on data partition via overlay mechanism".
at least I was a bit confused about reading only about "mnt" and "data" partitions here where the example used home directories (what is a perfectly valid application of this mechanism imo), so I felt it could be worth a little note about the background.

Reply:
so the problem is the initial documentation? is it better to remove the existing hints then?